RFE: Improve log when admitting next gen routes

#### Title
Title of the feature/enhancement request

#### Description

Submitting a 3 Next Gen Routes I found the following behaviour:

```
2024/02/01 15:06:26 [DEBUG] [AS3] posting request to https://10.1.1.7/mgmt/shared/appsvcs/declare/mc-onetier
2024/02/01 15:06:37 [DEBUG] [AS3] Response from BIG-IP: code: 200 --- tenant:mc-onetier --- message: success
2024/02/01 15:06:37 [DEBUG] [CORE] Admit Status Erased for Route - nginx-app1-reencrypt-xxx
2024/02/01 15:06:37 [DEBUG] [CORE] Admit Status Erased for Route - nginx-app1-xxx
2024/02/01 15:06:37 [DEBUG] [CORE] Admit Status Erased for Route - nginx-app1-edge-xxx
2024/02/01 15:06:37 [ERROR] Error while Updating Route Admit Status: Operation cannot be fulfilled on routes.route.openshift.io "nginx-app1-reencrypt-xxx": the object has been modified; please apply your changes to the latest version and try again
2024/02/01 15:06:37 [ERROR] Error while Updating Route Admit Status: Operation cannot be fulfilled on routes.route.openshift.io "nginx-app1-xxx": the object has been modified; please apply your changes to the latest version and try again
2024/02/01 15:06:37 [ERROR] Error while Updating Route Admit Status: Operation cannot be fulfilled on routes.route.openshift.io "nginx-app1-edge-xxx": the object has been modified; please apply your changes to the latest version and try again
2024/02/01 15:06:37 [DEBUG] Admitted Route -  nginx-app1-reencrypt-xxx
2024/02/01 15:06:37 [DEBUG] Admitted Route -  nginx-app1-xxx
2024/02/01 15:06:37 [DEBUG] Admitted Route -  nginx-app1-edge-xxx
```

When running CIS in normal INFO log level, the customer might think that the routes haven´t been admitted properly because will only see the ERROR logs and not the later Admited Route messages

#### Actual Problem

1) It is confusing that in the same second there is the error and the Route admitted
2) the Admitted Route message should be in INFO level

--- a/docs/RELEASE-NOTES.rst
+++ b/docs/RELEASE-NOTES.rst
@@ -22,6 +22,7 @@ Bug Fixes
 * `Issue 3230 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3230>`_: CRD multicluster configuration triggers Raw response from Big-IP: map[code:422 declarationFullId: message:declaration has duplicate values in rules].
 * `Issue 3239 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3239>`_: Support for mix of k8s Secret and bigip reference in TLSProfile.
 * `Issue 3232 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3232>`_: Enhance as3 response add the runtime attribute
+* `Issue 3266 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3266>`_: Improve log when admitting next gen routes
 * Fix for wildcard domain with multiple hosts in tls profile
 
 2.15.1

--- a/pkg/controller/nativeResourceWorker.go
+++ b/pkg/controller/nativeResourceWorker.go
@@ -1556,22 +1556,22 @@ func (ctlr *Controller) updateRouteAdmitStatus(
 		}
 		Admitted := false
 		now := metaV1.Now().Rfc3339Copy()
+		var routeStatusIngress []routeapi.RouteIngress
 		for _, routeIngress := range route.Status.Ingress {
 			if routeIngress.RouterName == F5RouterName {
 				for _, condition := range routeIngress.Conditions {
 					if condition.Status == status {
 						Admitted = true
-					} else {
-						// remove all multiple route admit status submitted earlier
-						ctlr.eraseRouteAdmitStatus(rscKey)
 					}
 				}
+			} else {
+				routeStatusIngress = append(routeStatusIngress, routeIngress)
 			}
 		}
 		if Admitted {
 			return
 		}
-		route.Status.Ingress = append(route.Status.Ingress, routeapi.RouteIngress{
+		routeStatusIngress = append(routeStatusIngress, routeapi.RouteIngress{
 			RouterName: F5RouterName,
 			Host:       route.Spec.Host,
 			Conditions: []routeapi.RouteIngressCondition{{
@@ -1582,9 +1582,11 @@ func (ctlr *Controller) updateRouteAdmitStatus(
 				LastTransitionTime: &now,
 			}},
 		})
+		// updating to the new status
+		route.Status.Ingress = routeStatusIngress
 		_, err := ctlr.routeClientV1.Routes(route.ObjectMeta.Namespace).UpdateStatus(context.TODO(), route, metaV1.UpdateOptions{})
 		if err == nil {
-			log.Debugf("Admitted Route -  %v", route.ObjectMeta.Name)
+			log.Infof("Admitted Route -  %v", route.ObjectMeta.Name)
 			return
 		}
 		log.Errorf("Error while Updating Route Admit Status: %v\n", err)

